src/main/java/javaracingcarwinner/entity/MoveStrategy.java

```diff
@@ -0,0 +1,5 @@
+package javaracingcarwinner.entity;
+
+public interface MoveStrategy {
+    int movingValue();
+}
```

Comment:
```suggestion
@FunctionalInterface
public interface MoveStrategy {
    int movingValue();
}
```
추상화 👍 

아래 테스트에서 함수형 인터페이스를 활용한 부분이 보이는데, 생략가능한 어노테이션이나 함수형 인터페이스로 활용하시게되는 경우 가능하면 명시해주시는 것이 좋습니다 😄